db/migrate/20200428163027_rename_title_to_subject_in_notes.rb

```diff
@@ -0,0 +1,7 @@
+class RenameTitleToSubjectInNotes < ActiveRecord::Migration[6.0]
+  def change
+    change_table :notes do |t|
+      t.rename :title, :subject
```

Comment:
So glad we had this conversation now rather than later! 🎉

Reply:
> Teensy UI consideration, probably not for this PR: when I click "Note added" in the Timeline, I return to the notes index with an anchor, which is great. But the page is short enough (and probably will be, up to a dozen or so notes) that I can't tell which note I've clicked on. Should we highlight the anchored note or sth? I am not a designer.

I think @adamsilver is planning something for this. I've looked into highlighting the relevant note in the view, but rails cannot retrieve the anchor part of the url, so it'd need to be something javascript-driven. Probably best tackled when the designs are revised.